flit_core/tests_core/test_wheel.py

```diff
@@ -45,3 +46,16 @@ def test_data_dir(tmp_path):
     assert_isfile(info.file)
     with ZipFile(info.file, 'r') as zf:
         assert 'module1-0.1.data/data/share/man/man1/foo.1' in zf.namelist()
+
+
+def test_license_files(tmp_path):
+    dir = os.getcwd()
+    try:
+        os.chdir(samples_dir / 'pep621_license_files')
```

Comment:
I don't think we need to modify the CWD here - e.g. the test just above doesn't do this.